templates/testdata/RELEASE_NOTES_unified.md

```diff
@@ -19,6 +19,23 @@ to their releases:
 
 ## Upgrade Instructions
 
+### Conjur
+
+Upgrading Conjur generally requires modifying the container image tag to the value in the release suite. Below are more specific instructions depending on environment.
+
+For running on **Docker**, e.g. in the [quickstart docker-compose.yml](https://github.com/cyberark/conjur-quickstart/blob/master/docker-compose.yml) change the container image tag to:
+```
+image: cyberark/conjur:1.4.4
+```
+
+For [**Cloud Formation templates for AWS**](https://github.com/cyberark/conjur-aws) set the environment variable CONJUR_VERSION before building the AMI:
```

Comment:
I'd put the `^For` items into bulleted points with the type bolded and first on the line for easier reading